serendipity/settings.py

```diff
@@ -38,7 +38,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'users.apps.UsersConfig',
+    'todo_list.apps.TodoListConfig'
```

Comment:
@quratulain25 I think this app was directly added to master branch so I can't review TodoListConfig